src/stores/useHoloStore.js

```diff
@@ -121,6 +118,15 @@ const makeUseHoloStore = ({ connectionArgs, MockWebSdk }) => defineStore('holo',
       this.kycLevel = kycLevel
       return kycLevel
     },
+    async loadJurisdictions(environment, hbsServicePort) {
+      try {
+        registrationFetchJurisdictions(environment, hbsServicePort).then((jurisdictions) => {
+          this.jurisdictions = jurisdictions.data
+        })
+      } catch (e) {
+        console.error(`Error fetching jurisdictions: ${e}`)
+      }
+    },    
```

Comment:
I think it could be time to move this and kycLevel into their own store, distinct from `holoStore`.

Something like `hbsStore` might make sense.

I'll leave it up to you if you want to do this in this PR, but just realizing that all of this hbs stuff is scope creep from what `holoStore` was originally intended for